compiler/test/compilable/imports/defines.c

```diff
@@ -30,3 +30,6 @@ _Static_assert(SSS[0] == 'h', "10");
 #define ABC 12
 #define GHI (size) abbadabba
 #define DEF (ABC + 5)
+
+#define ADD(a, b) a + b
+#define SUB() 3 - 2
```

Comment:
What about some of empty body tests?

Not to forget, for some reason the most common style of function-like macro I've come across is `do{ multiple; things; } while(0)`
```
#define NO_BODY()
#define NO_BODY_PARAMS(a, b)
#define BODY_PARENS() ()
#define BODY_BRACES() {}
#define DO_WHILE() do { } while(0)
```

Reply:
See https://hownot2code.wordpress.com/2016/12/05/do-while-0-in-macros/

Reply:
I'll do statements in a later PR after this one is pulled. This one only does expressions.